python/cugraph/pagerank/test_pagerank.py

```diff
@@ -80,6 +91,19 @@ def networkx_call(M, max_iter, tol, alpha):
     if M.shape[0] != M.shape[1]:
         raise TypeError('Shape is not square')
 
+    personalization = None
+    if personalization_perc != 0:
```

Comment:
1. We should explain how the personalization vector is set. I'm not completely sure about what's in there in the end.
2. In cyber, some users don't need values. They just have a set of vertex they want to parametrize. We would need to add a test for that where `val[i] = 1.0/num_personalized_vertices` if `i` is parametrized and `0.0` otherwise.

Reply:
1. Fixed [here](https://github.com/rapidsai/cugraph/pull/300/files?file-filters%5B%5D=.h&file-filters%5B%5D=.md&file-filters%5B%5D=.py&file-filters%5B%5D=.sh&owned-by%5B%5D=#diff-83a484342317d80b4509f185eee9a5bbR85)
2. That case is already tested in personalization=None